velox/functions/lib/aggregates/tests/utils/AggregationTestBase.cpp

```diff
@@ -959,6 +959,70 @@ void AggregationTestBase::testAggregationsImpl(
   }
 }
 
+void AggregationTestBase::testStreamingAggregationsImpl(
+    std::function<void(PlanBuilder&)> makeSource,
+    const std::vector<std::string>& groupingKeys,
+    const std::vector<std::string>& aggregates,
+    const std::vector<std::string>& postAggregationProjections,
+    std::function<std::shared_ptr<exec::Task>(AssertQueryBuilder&)>
+        assertResults,
+    const std::unordered_map<std::string, std::string>& config) {
+  {
+    SCOPED_TRACE("Test StreamingAggregation: run single");
```

Comment:
nit: "Streaming: run single" might be sufficient, both "Test" and "Aggregation" are implied.